conf/application.conf

```diff
@@ -42,7 +42,7 @@ play.modules.enabled += "timetopaytaxpayer.cor.TaxpayerCorModule"
 mongodb.uri = "mongodb://localhost:27017/self-service-time-to-pay-frontend"
 # Journey TTL must be set in excess of expected user journey time because timeout implementation doesn't reset the TTL
 # Journey TTL is set to 2000 days to align with the default test data (FROZEN DATE) in the stubs for the test-only endpoint
-journey.ttl = 2000 days
+journey.ttl = 1.days
```

Comment:
the comment just above doesn't seem valid anymore if it's not 2000 days anymore

Reply:
removed